.gitignore

```diff
@@ -6,6 +6,15 @@
 target
 work
 *.iml
-yet-another-docker-java/dependency-reduced-pom.xml
+.*.swp
+.swp
 /target
 
+yet-another-docker-java/dependency-reduced-pom.xml
+yet-another-docker-java/.classpath
+yet-another-docker-java/.project
+yet-another-docker-plugin/.classpath
+yet-another-docker-plugin/.project
+yet-another-docker-plugin/.settings/org.eclipse.jdt.core.prefs
+yet-another-docker-plugin/.settings/de.loskutov.anyedit.AnyEditTools.prefs
```

Comment:
is it eclipse ignore rules? maybe ignore with `*/.classpath` ? 
https://github.com/github/gitignore/commit/0674ff5b5c77935cbb2ea2269c001bfd84aa1aa5 hm..